src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  todo?: Todo | null,
+  onSelect: (todo: Todo | null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onSelect = () => {} }) => {
+  const [modalLoading, setModalLoading] = useState(false);
```

Comment:
```suggestion
  const [modalLoading, setModalLoading] = useState(true);
```
to remove blinks when you're opening the modal for the first time